SampleApps/WebView2APISample/README.md

```diff
@@ -1,14 +1,30 @@
+---
+description: "Demonstrate the features and usage patterns of WebView2 in Win32."
```

Comment:
These descriptions get stripped out before end-users see the docs right?

Reply:
Correct -- it is shown in the search results. You can see [here](https://review.docs.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=master#yaml-front-matter-structure) for more details.
![image](https://user-images.githubusercontent.com/55165503/90082537-dd234b80-dcc4-11ea-93b0-73468146e7f8.png)